roles/etcd_client_certificates/tasks/main.yml

```diff
@@ -1,4 +1,24 @@
 ---
+- name: DEBUG etcd_cert_config_dir
```

Comment:
@djosborne not sure if you caught this but you brought over some of my debug / pause actions.